deployments/deployment.yaml

```
@@ -576,7 +577,7 @@ parameters:
   - name: IMAGE
     value: quay.io/cloudservices/content-sources-backend
   - name: NIGHTLY_CRON_JOB
-    value: "0 0/8 * * *"
+    value: "0 0/24 * * *"
```

```suggestion
    value: "0 0/1 * * *"
```
I think it would need to be 1 to run once an hour, right?

---

hah, good call!